docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md

```diff
@@ -101,26 +101,26 @@ The [YugabyteDB Tablet Server](../../../architecture/concepts/yb-tserver/) (YB-T
 | :---| :--- | :--- |
 | YB-TServer Operations/sec | The number of [YB-TServer](../../../architecture/concepts/yb-tserver/) read and write operations per second. | Spikes in read operations are normal during backups and scheduled maintenance. If the count drops significantly below average, it might indicate an application connection failure. If the count is much higher than average, it could indicate a DDoS, security incident, and so on. Coordinate with your application team because there could be legitimate reasons for dips and spikes. |
 | YB-TServer Average Latency (ms) | Read: the average latency of read operations at the tablet level.<br>Write: the average latency of write operations at the tablet level. | When latency starts to degrade, performance may be impacted by the storage layer. |
-| Handler Latency (ms) | Incoming queue: The number of microseconds the incoming RPC requests spend in the worker queue before the beginning of processing.<br>Outbound queue time: The number of microseconds between an outbound call being created and put in the queue.<br>Outbound transfer time: The number of microseconds the outgoing traffic takes to exit the queue.<br>Note that Handler is a software implementation of a ring queue. | If this metric spikes or remains at a high level, it indicates a network issue or that the queues are full. |
+| Handler Latency (ms) | Incoming queue: time (in milliseconds) incoming RPC requests spend in the worker queue before the beginning of processing.<br>Outbound queue time: time (in milliseconds) between an outbound call being created and put in the queue.<br>Outbound transfer time: time (in milliseconds) the outgoing traffic takes to exit the queue.<br>Note that Handler is a software implementation of a ring queue. | If this metric spikes or remains at a high level, it indicates a network issue or that the queues are full. |
```

Comment:
Line 74 : Network Bytes/Sec (MiB) - mentions bytes.. this also needs to be changed to mebibytes. Also line 75.